src/models/WeatherDetails.tsx

```diff
@@ -0,0 +1,14 @@
+export default class WeatherDetails {
+
+    id: number;
+    main: string;
+    description: string;
+    icon: string;
```

Comment:
I suppose we are holding the icon path, and not the icon code itself, is that right? If so, I'd suggest renaming this attribute name to `iconPath`.

Reply:
It seems that we don't need this attribute. Removed.